jwala-integration-test/src/test/java/com/cerner/jwala/ui/selenium/steps/CommonRunSteps.java

```diff
@@ -311,22 +312,22 @@ public void deleteLine(Map<String, String> parameters) {
         handleResourceRunSteps.clickSaveButton(parameters.get("tabLabel"));
     }
 
-    @And("^I enter attribute in the file MetaData with the following parameters:$")
-    public void enterAttributeInMetaData(Map<String, String> parameters) {
-        navigationRunSteps.goToConfigurationTab();
-        navigationRunSteps.goToResourceTab();
-        uploadResourceRunSteps.expandNode(parameters.get("group"));
-        uploadResourceRunSteps.expandNode(parameters.get("componentType"));
-        uploadResourceRunSteps.clickNode(parameters.get("componentName"));
-        handleResourceRunSteps.clickResource(parameters.get("fileName"));
-        handleResourceRunSteps.clickTab("Meta Data");
-        handleResourceRunSteps.enterAttribute(parameters.get("attributeKey"), parameters.get("attributeValue"));
-        handleResourceRunSteps.clickSaveButton("Meta Data");
-        if(parameters.get("override").equals("true")){
-            jwalaUi.clickOkWithSpan();
-        }
-        handleResourceRunSteps.waitForNotification("Saved");
-    }
+	@And("^I enter attribute in the file MetaData with the following parameters:$")
```

Comment:
It's difficult to tell if there are differences here or not because of the formatting